### PR TITLE
[Microsoft.ResourceGraph] Returning null for nextLink in example to preserve the existing scenario

### PR DESCRIPTION
**Pull request by ericc1103**
### Latest improvements:
<i>MSFT employees can try out our new experience at <b>[OpenAPI Hub](https://aka.ms/openapiportal) </b> - one location for using our validation tools and finding your workflow. 
</i><br>
### Contribution checklist:
- [ ] I have reviewed the [documentation](https://github.com/Azure/azure-rest-api-specs#basics) for the workflow.
- [ ] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.
- [ ] The [OpenAPI Hub](https://aka.ms/openapiportal) was used for checking validation status and next steps.
### ARM API Review Checklist
- [ ] Service team MUST add the "WaitForARMFeedback" label if the management plane API changes fall into one of the below categories. 
- adding/removing APIs.
- adding/removing properties.
- adding/removing API-version. 
- adding a new service in Azure.

Failure to comply may result in delays for manifest application. Note this does not apply to data plane APIs.
- [ ] If you are blocked on ARM review and want to get the PR merged urgently, please get the ARM oncall for reviews (RP Manifest Approvers team under Azure Resource Manager service) from IcM and reach out to them. 
Please follow the link to find more details on [API review process](https://armwiki.azurewebsites.net/rp_onboarding/ResourceProviderOnboardingAPIRevieworkflow.html).

### CODE REVIEW

File: specification/resourcegraph/resource-manager/Microsoft.ResourceGraph/preview/2018-09-01-preview/examples/OperationsList.json [hunk 1]
```diff
@@ -5,7 +5,7 @@
   "responses": {
     "200": {
       "body": {
-        "nextLink": "https://management.azure.com/subscriptions/subscriptionId/providers/Microsoft.ResourceGraph/?api-version=2019-09-01-preview&$top=10&$skiptoken=skiptoken",
```
Comment (jhendrixMSFT) [inline specification/resourcegraph/resource-manager/Microsoft.ResourceGraph/preview/2018-09-01-preview/examples/OperationsList.json]: Is the actual response a "nextLink" key with null value, or should this line be omitted entirely?

Comment (ericc1103) [reply specification/resourcegraph/resource-manager/Microsoft.ResourceGraph/preview/2018-09-01-preview/examples/OperationsList.json]: Adding @yangmsft

Comment (yangmsft) [reply specification/resourcegraph/resource-manager/Microsoft.ResourceGraph/preview/2018-09-01-preview/examples/OperationsList.json]: If we can omit it and validation allows it, let's omit the line.

**Comment by jhendrixMSFT** [comment]
@ericc1103 there are a lot of commits showing up, I think your fork needs to be updated.

**Comment by ericc1103** [comment]
> @ericc1103 there are a lot of commits showing up, I think your fork needs to be updated.

Hmmm, I have updated but it still shows. This is what I did:
` git fetch upstream -v`
`git merge upstream/master`

**Review by jhendrixMSFT — APPROVED** [review]

**Pull request merged**